docs/new_viz_guidance.md

```diff
@@ -35,14 +35,13 @@ Database are in the folder /queries/. These queries may be reused across multipl
 visualizations. When considering creating your own visualization, please
 see whether the available queries to our database instance are sufficient.
 
-If the existing queries won't suffice, the most straight-forward way to create
-your own is to copy one of the existing queries into the same directory with a
-descriptive filename:
+If the existing queries do not suffice:
+1. See if there is some minor changes or additional columns that could be added to
+an existing query to get the necessary data.
+2. If a completely new query is neccessary, use the [Query Template](https://github.com/oss-aspen/8Knot/blob/dev/queries/query_template.py)
+to build a new query.
 
-e.g. "mv commits\_query.py ./comments\_query.py"
-
-Then, change as few things as possible in the newly created file to meet the
-needs of your new query. Make sure to include comments.
+Note: make sure to do a "docker system prune -af" before rebuilding to be able to access any new or edited queries.
```

Comment:
Using 'docker compose down' is sufficient. 'docker system prune -af' will destroy EVERYTHING they're running, including any other docker services or volumes.